Report/Chapters/Process.tex

```diff
@@ -28,17 +28,17 @@ \section{Code tools}
 
 \subsection{Maven}
 
-Larger Java projects usually have dependencies. These dependencies can also have dependencies. To make sure every team member was using the same versions of the libraries the project was no doubt going to use, we decided to use Maven, a Java build tool. Using Maven as a standardized build process also allowed team members to work from their preferred environment: the back end crew preferred IntelliJ IDEA by Jetbrains, while the front end developers worked mostly in Eclipse. \index{Maven}
+Larger Java projects usually have dependencies. These dependencies can also have dependencies. To make sure every team member was using the same versions of the libraries the project was no doubt going to use, we decided to use Maven, a Java build tool. Using Maven as a standardized build process also allowed team members to work from their preferred environment: the back end crew preferred IntelliJ IDEA by JetBrains, while the front end developers worked mostly in Eclipse. \index{Maven}
 
 \subsection{Git, GitHub and GitHub Flow}
 
 For the version control system, we decided on git, with the central repository stored on GitHub. \index{git} \index{GitHub}
 
-As for a branching model, we decided on GitHub Flow\cite{githubflow}. In this model, small, independent changes are published to their own branches, tested, checked, then proposed as a Pull Request. At this point the person responsible for code integration (either Wander or Jan-Jelle) reviews the code and merge it into the master branch. \index{GitHub Flow}
+As for a branching model, we decided on GitHub Flow\cite{githubflow}. In this model, small, independent changes are published to their own branches, tested, checked, then proposed as a Pull Request. At this point the person responsible for code integration (either Wander or Jan-Jelle) reviews the code and merge it into the master branch. \index{GitHub Flow} Additionally, GitHub is used as issue tracker.
```

Comment:
Additionally, GitHub *was* used as issue tracker. ?
Not 100% sure  what time you are using, but I have a feeling this is inconsitent with your previous usage of time.